docs/pages/docs/01-cloud-deployment/best-practices.md

```diff
@@ -8,27 +8,52 @@ permalink: /docs/cloud-deployment/best-practices
 folder: docs
 ---
 
-# GC Deployment Best Pracices
+# GC Deployment Best Practices
 
 In this section we shall describe the recommendations for comput and storage aspects of GC deployments
```

Comment:
the recommendations for _computational/compute_ and storage aspects

Is compute the right name? Don't know what Google uses